internal/definitions/sender_contracts.go

```diff
@@ -55,17 +55,17 @@ func (bm *definitionSender) DefineContractAPI(ctx context.Context, httpServerURL
 	api.ID = fftypes.NewUUID()
 
 	if bm.multiparty {
-		if err := bm.contracts.ResolveContractAPI(ctx, httpServerURL, api); err != nil {
```

Comment:
I think we still want to do the resolution before broadcasting the definition... but perhaps we need to set the namespace, then clear it before the broadcast, then set it again before we return...

Reply:
The other solution I was proposing was to just use the `ContractManager` namespace here: https://github.com/hyperledger/firefly/blob/1f31807dd98c22f45f00528cebb26071d5b25457/internal/contracts/manager.go#L257

Reply:
Yea, that seems like a fine solution